src/system/AdminModule/Controller/AdminInterfaceController.php

```diff
@@ -0,0 +1,579 @@
+<?php
+/**
+ * Copyright Zikula Foundation 2009 - Zikula Application Framework
+ *
+ * This work is contributed to the Zikula Foundation under one or more
+ * Contributor Agreements and licensed to You under the following license:
+ *
+ * @license GNU/LGPLv3 (or at your option, any later version).
+ *
+ * Please see the NOTICE file distributed with this source code for further
+ * information regarding copyright and licensing.
+ */
+
+namespace Zikula\AdminModule\Controller;
+
+use Symfony\Component\Security\Core\Exception\AccessDeniedException;
+use Symfony\Component\HttpFoundation\Response;
+use Sensio\Bundle\FrameworkExtraBundle\Configuration\Route; // used in annotations - do not remove
+use Sensio\Bundle\FrameworkExtraBundle\Configuration\Method; // used in annotations - do not remove
+use Sensio\Bundle\FrameworkExtraBundle\Configuration\Template;
+use Zikula\Core\Controller\AbstractController;
+use Zikula\Bundle\CoreBundle\Console\Application;
+use Zikula\Core\Theme\Annotation\Theme;
+
+
+/**
+ * @Route("/admininterface")
+ */
+class AdminInterfaceController extends AbstractController
+{	
+	/**
+	 * @Route("/header")
+	 *
+	 * @Template
+	 * @Theme("admin")
+	 *
+	 * Open the admin container
+	 *
+	 * @return Response symfony response object
+	 */
+	public function headerAction()
+	{
+		$masterRequest = $this->get('request_stack')->getMasterRequest();
+		$caller = array();
+		$caller['_zkModule'] = $masterRequest->attributes->get('_zkModule');
+		$caller['_zkType'] = $masterRequest->attributes->get('_zkType');
+		$caller['_zkFunc'] = $masterRequest->attributes->get('_zkFunc');	 
+		return ['caller' => $caller];
+	}
+	
+	/**
+	 * @Route("/footer")
+	 *
+	 * @Template
+	 * @Theme("admin")
+	 *
+	 * Close the admin container
+	 *
+	 * @return Response symfony response object
+	 */
+	public function footerAction()
+	{
+		$masterRequest = $this->get('request_stack')->getMasterRequest();
+		$caller = array();
+		$caller['_zkModule'] = $masterRequest->attributes->get('_zkModule');
+		$caller['info'] = \ModUtil::getInfoFromName($caller['_zkModule']);	
+		return ['caller' => $caller,'symfonyversion' => \Symfony\Component\HttpKernel\Kernel::VERSION, 'phpversion' => phpversion()];
+	}
+	
+	/**
+	 * @Route("/help")
+	 *
+	 * @Template
+	 * @Theme("admin")
+	 *
+	 * display the module help page
+	 *
+	 * @return Response symfony response object
+	 *
+	 * @throws AccessDeniedException Thrown if the user doesn't have admin permission to the module
+	 */
+	public function helpAction()
```

Comment:
This function does nothing now, it was actually ZikulaAdminModule help function... copied. https://github.com/zikula/core/blob/1.4/src/system/AdminModule/Resources/views/Admin/help.tpl

I can make this function to read and display twig/.md file in caller directory for example ```{{caller._zkModule}}\Resources\public\docs\admin\help...```
It will be placed in admin header and show link to caller specific admin help will be called 
{{render (controller(ZikulaAdminModule:AdminInterface:help))}}